lib/view/pages/home_screen.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/material.dart';
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:iti_final_project/resources/constants/app_constants.dart';
+import '../widgets/my_text.dart';
+import '../widgets/product_card.dart';
+import '../widgets/search_field.dart';
+import '../widgets/section_header.dart';
+
```

Comment:
- you can put sliderImages list & popularCars list  & bestSeling list in app_constants.dart

Reply:
Done